app/src/main/res/layout/fragment_plant_list.xml

```diff
@@ -15,17 +15,33 @@
   ~ limitations under the License.
   -->
 
-<layout xmlns:android="http://schemas.android.com/apk/res/android">
-    <android.support.v7.widget.RecyclerView
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
         xmlns:app="http://schemas.android.com/apk/res-auto"
-        xmlns:tools="http://schemas.android.com/tools"
-        android:id="@+id/plant_list"
-        android:name="com.google.samples.apps.sunflower.PlantListFragment"
+        xmlns:tools="http://schemas.android.com/tools">
+
+    <FrameLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_marginLeft="@dimen/margin_normal"
-        android:layout_marginRight="@dimen/margin_normal"
-        app:layoutManager="android.support.v7.widget.LinearLayoutManager"
-        tools:context="com.google.samples.apps.sunflower.GardenActivity"
-        tools:listitem="@layout/list_item_plant" />
+        android:paddingLeft="@dimen/margin_normal"
+        android:paddingRight="@dimen/margin_normal"
+        android:layout_height="match_parent">
+
+        <android.support.v7.widget.RecyclerView
+            android:id="@+id/plant_list"
+            android:name="com.google.samples.apps.sunflower.PlantListFragment"
```

Comment:
Shouldn't `android:name` be added to parent, I mean to `FrameLayout` after your layout changes?

Reply:
Sorry you are right.

Reply:
@XinyueZ I noticed also that in the second MR you have the same "issue"